puppet-profiles/springboot/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Same question

Reply:
File created by the PDK convert command, no idea what's used for except making the Puppet module compliant... ;-)